ios-quickeditor/ios-quickeditor/QEFilesListViewController.m

```diff
@@ -41,16 +36,14 @@ - (void)awakeFromNib
 
 - (void)viewDidLoad
 {
-    [super viewDidLoad];
-    
-    // Check for authorization.
-    GTMOAuth2Authentication *auth =
-    [GTMOAuth2ViewControllerTouch authForGoogleFromKeychainForName:kKeychainItemName
-                                                          clientID:kClientId
-                                                      clientSecret:kClientSecret];
-    if ([auth canAuthorize]) {
-        [self isAuthorizedWithAuthentication:auth];
-    }
+    // Configure Google Sign-in.
+    GIDSignIn* signIn = [GIDSignIn sharedInstance];
+    signIn.delegate = self;
```

Comment:
From what I've gathered, it's not a strong requirement/recommendation. Other docs/samples contradict it (e.g https://developers.google.com/identity/sign-in/ios/quick-migration-guide shows it on a ViewController instead of AppDelegate.

Can move it there, but it adds complexity in that the event would just have to be rebroadcast to the ViewController anyway.